src/Microsoft.Diagnostics.Monitoring.Egress.AzureStorage/AzureBlobEgressStreamOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Diagnostics.Monitoring.Egress.AzureStorage
+{
+    internal class AzureBlobEgressStreamOptions
+    {
+        public string ContentType { get; set; }
```

Comment:
Could you also add `ContentEncoding` ?
This would be used if the client does gzip compression during upload (the ContentEncoding is then set to `gzip`)

Reply:
Added